traffic_ops/app/db/migrations/2022092107561215_ds_explicit_mso.down.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+ALTER TABLE public.deliveryservice
```

Comment:
The up.sql file has updates on `deliveryservice_request` table but I see nothing to revert that in down file?

Reply:
Those changes don't need to be reverted. Before the change, `multiSiteOrigin` can be one of three values: `true`, `false`, or `null`. The `up` file changes all `null` values to `false` (because `null` and `false` are treated exactly the same, so there was no reason to allow both). The `up` file needed to do that because after it runs, `null` will no longer be allowed. But after `down` runs, `false` is still allowed - and also it's treated the same as `null` so there's no behavioral impact, and finally you also can't know which `false`s used to be `null`s and which were `false` in the first place, so the information needed to restore that state doesn't exist anyway.